task/prefetch-dependencies/0.1/prefetch-dependencies.yaml

```diff
@@ -188,6 +188,48 @@ spec:
         echo "$input"
       }
 
+      function inject_rpm_summary_flag
+      {
+        input="$1"
+
+        if [ "$input" == "rpm" ]; then
+          input="$(jq -n '{type: "rpm", include_summary_in_sbom: "true"}')"
```

Comment:
JSON supports booleans, so we should probably format boolean fields as such instead of strings.